client/blocks/comment-detail/style.scss

```diff
@@ -12,9 +12,16 @@
 		}
 	}
 
-	&.is-collapsed:hover {
-		box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20% );
-		z-index: 1;
+	&.is-collapsed {
+		&:hover {
+			box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20% );
+			z-index: 1;
+		}
+
+		.accessible-focus &:focus {
+			box-shadow: 0 0 0 1px $blue-medium, 0 0 0 3px $blue-light;
+			z-index: 1;
```

Comment:
This might originally be my fault though 😨 
IIRC, the z-index is needed so that the bottom box shadow (here used instead of border) doesn't appear behind the next card (which is what we want when both cards have the same box shadow color, but not when one of them is highlighted).
Not sure if this is granted a slot in the `_z-index.scss`, it being a workaround.